src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+import { InputBase, TextField } from "@material-ui/core";
+import { Grid } from "@material-ui/core";
+import { useMemo } from "react";
+import { fetchByTags } from "../../lib";
+import _ from "lodash";
```

Comment:
does lodash allow you to only import the method you need to use? like `"lodash/debounce` ? its a pretty big library and it would be cool to avoid importing the whole thing

Reply:
looks like you can: https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark